data/json/items/melee/spears_and_polearms.json

```
@@ -348,6 +348,28 @@
     "use_action": [ "HEAT_SOLID_ITEMS" ],
     "melee_damage": { "bash": 6, "stab": 30 }
   },
+  {
+    "type": "ITEM",
+    "id": "spear_machined",
+    "name": { "str": "machined steel spear" },
+    "description": "A stout wooden pole with a cylindrical steel spearhead that was turned on a lathe.",
+    "weight": "1600 g",
+    "price_postapoc": "55 USD",
+    "color": "light_gray",
+    "symbol": "/",
+    "material": [ "steel", "wood" ],
+    "techniques": [ "WBLOCK_1", "PENETRATE" ],
+    "volume": "1350 ml",
+    "longest_side": "180 cm",
+    "to_hit": { "grip": "weapon", "length": "long", "surface": "point", "balance": "good" },
+    "flags": [ "DURABLE_MELEE", "SPEAR", "REACH_ATTACK", "NONCONDUCTIVE", "SHEATH_SPEAR" ],
+    "faults": [ { "fault_group": "handle_long", "weight_mult": 4 }, { "fault_group": "spearhead" } ],
+    "weapon_category": [ "SPEARS" ],
+    "price": "160 USD",
+    "qualities": [ [ "COOK", 1 ] ],
+    "use_action": [ "HEAT_SOLID_ITEMS" ],
+    "melee_damage": { "bash": 4, "stab": 32 }
```

A conical spear head would be fairly terrible damage wise, there's a reason they're made as flat blades. 
```suggestion
    "melee_damage": { "bash": 4, "stab": 20 }
```

---

i agree theyd be worse than a broadhead style spear but i dont think itd be that bad. im thinking itd either be similar to a mosin nagant bayonet, or the tip of a giant nail